cypress/support/step-definitions/date-input-steps.js

```diff
@@ -11,22 +11,22 @@ const TODAY_KNOBS = Cypress.moment().format('YYYY-MM-DD');
 const TODAY_DATE_INPUT = Cypress.moment().format('DD/MM/YYYY');
 
 Then('Date input is disabled', () => {
-  dateInput().parent()
+  dateInputNoIFrame().parent()
```

Comment:
could we move `parent()` to index file, if we are not using simple `dateInputNoIFrame()` without it

Reply:
fixed